src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -66,50 +64,37 @@
 import java.util.Set;
 import org.hamcrest.core.IsNot;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.spongycastle.util.encoders.Hex;
 
 public class ScriptTest {
     // From tx 05e04c26c12fe408a3c1b71aa7996403f6acad1045252b1c62e055496f4d2cb1 on the testnet.
-    static final String sigProg = "47304402202b4da291cc39faf8433911988f9f49fc5c995812ca2f94db61468839c228c3e90220628bff3ff32ec95825092fa051cba28558a981fcf59ce184b14f2e215e69106701410414b38f4be3bb9fa0f4f32b74af07152b2f2f630bc02122a491137b6c523e46f18a0d5034418966f93dfc37cc3739ef7b2007213a302b7fba161557f4ad644a1c";
-    static final String pubkeyProg = "76a91433e81a941e64cda12c6a299ed322ddbdd03f8d0e88ac";
+    private static final String scriptSig = "47304402202b4da291cc39faf8433911988f9f49fc5c995812ca2f94db61468839c228c3e90220628bff3ff32ec95825092fa051cba28558a981fcf59ce184b14f2e215e69106701410414b38f4be3bb9fa0f4f32b74af07152b2f2f630bc02122a491137b6c523e46f18a0d5034418966f93dfc37cc3739ef7b2007213a302b7fba161557f4ad644a1c";
+    private static final String pubkeyProg = "76a91433e81a941e64cda12c6a299ed322ddbdd03f8d0e88ac";
+    private static final String expectedAddressFromScriptSig = "15jTWe6r9zqxkjjLFntAWADZosAwiuw4U5";
 
-    private static final NetworkParameters PARAMS = TestNet3Params.get();
+    private static final NetworkParameters mainnetParams = MainNetParams.get();
     private static final Logger log = LoggerFactory.getLogger(ScriptTest.class);
-    private final List<BtcECKey> btcECKeyList = new ArrayList<>();
-    private final List<BtcECKey> erpFedECKeyList = new ArrayList<>();
-    private final BtcECKey ecKey1 = BtcECKey.fromPrivate(BigInteger.valueOf(100));
-    private final BtcECKey ecKey2 = BtcECKey.fromPrivate(BigInteger.valueOf(200));
-    private final BtcECKey ecKey3 = BtcECKey.fromPrivate(BigInteger.valueOf(300));
-    private final BtcECKey ecKey4 = BtcECKey.fromPrivate(BigInteger.valueOf(400));
-    private final BtcECKey ecKey5 = BtcECKey.fromPrivate(BigInteger.valueOf(500));
-    private final BtcECKey ecKey6 = BtcECKey.fromPrivate(BigInteger.valueOf(600));
-    private final BtcECKey ecKey7 = BtcECKey.fromPrivate(BigInteger.valueOf(700));
-    private final BtcECKey ecKey8 = BtcECKey.fromPrivate(BigInteger.valueOf(800));
-
-    @Before
-    public void setUp() {
-        btcECKeyList.add(ecKey1);
-        btcECKeyList.add(ecKey2);
-        btcECKeyList.add(ecKey3);
-        erpFedECKeyList.add(ecKey4);
-        erpFedECKeyList.add(ecKey5);
-        erpFedECKeyList.add(ecKey6);
-        erpFedECKeyList.add(ecKey7);
-        erpFedECKeyList.add(ecKey8);
-    }
+    private final List<BtcECKey> federationKeys = RedeemScriptUtils.getDefaultRedeemScriptKeys();
+
+    private final int redeemScriptStandardOpCodesNumberOfChunks = 2;
```

Comment:
Not sure what this name means

Reply:
This refers to the OP0 and RedeemScript chunks in a script sig. I updated this constant name.